opaque_keys/edx/keys.py

```diff
@@ -93,19 +93,19 @@ def make_asset_key(self, asset_type: str, path: str) -> AssetKey:  # pragma: no
         raise NotImplementedError()
 
 
-class LibraryCollectionKey(OpaqueKey):
+class LibraryItemKey(OpaqueKey):
     """
-    An :class:`opaque_keys.OpaqueKey` identifying a particular Library Collection object.
+    An :class:`opaque_keys.OpaqueKey` identifying a particular item in a library.
     """
-    KEY_TYPE = 'collection_key'
+    KEY_TYPE = 'library_element_key'
```

Comment:
If we're renaming this to `LibraryItemKey` instead of `LibraryElementKey`, shouldn't this key type be `library_item_key`?

Reply:
You are right. Updated: https://github.com/openedx/opaque-keys/pull/369/commits/75ecbe6e610e73e6d4def545106e5fb89356840e